packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -55,5 +86,22 @@ const Box = styled.div(
 ) as React.FC<BoxProps>;
 
 Box.displayName = 'Box';
-export {Box};
+
+const PseudoBox = styled(Box)(props => {
+  const pseudoProps = Object.keys(props).filter(propName => propName.startsWith('_'));
+  console.log(pseudoProps);
+  if (pseudoProps.length === 0) {
+    return {};
+  }
+
+  const pseudoStyles = {};
+  for (const pseudoProp of pseudoProps) {
+    pseudoStyles[PseudoStyles[pseudoProp.replace('_', '')]] = props[pseudoProp];
+  }
+
+  console.log(pseudoStyles);
+  return css(pseudoStyles);
```

Comment:
I should add comments but basically:
1. Find all props that start with a `_`, we will use this as our way to tell if the prop is a pseudo selector
2. Build an object of styles from those props where the key is retrieved from the PseudoStyles.tsx file, and the value is from the prop
3. Run this through a modified `@styled-system/css` so we can get the right token from a string value.